web/src/layers/ModalFilterLayer.svelte

```diff
@@ -3,6 +3,8 @@
   import { emptyGeojson } from "svelte-utils/map";
   import { layerId } from "../common";
   import { backend, mutationCounter } from "../stores";
+  // TODO Maybe make another component wrapping both of these
```

Comment:
re: comment - "both"? What's the other layer? Or maybe this is todone?

Reply:
Clarified. At first I was going to make every call-site include TurnRestrictionLayer, but it felt redundant, because it's all the same places as ModalFilterLayer. To be more pedantic, we could rename one of these or have some kind of `IconLayer`? But for now, I'm just punting on this